.github/actions/action-maven-release/action.yml

```
@@ -0,0 +1,91 @@
+        echo "MVN_ARTIFACT_ID=$MVN_ARTIFACT_ID" >> $GITHUB_OUTPUT
+        mvn release:prepare release:perform -f .${{inputs.app-path}}/pom.xml -B -DreleaseVersion=${{ inputs.releaseVersion }} -DdevelopmentVersion=${{ inputs.developmentVersion }} -Darguments="-Dmaven.deploy.skip=${{ inputs.skipDeployment }}"
+      env:
+        SIGN_KEY_PASS: ${{ inputs.GDP_PRIVATE_KEY }}
+        CENTRAL_USERNAME: ${{ inputs.CENTRAL_USERNAME }}
+        CENTRAL_PASSWORD: ${{ inputs.CENTRAL_PASSWORD }}
+
+    - id: artifact-name
+      run: echo "artifact-name=${{hashFiles(format('./{0}/pom.xml', inputs.app-path))}}" >> "$GITHUB_OUTPUT"
+      shell: bash
+    - name: "Upload Artifact"
+      uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4.6.0
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)